<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>S21Deposit</class>
 <widget class="QDialog" name="S21Deposit">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Deposit Calculator</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,1" columnstretch="5,9,0">
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="spinBox_tax">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>13.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="pushButton_plus_withdraw">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_capitalization">
      <property name="text">
       <string>Interest capitalization</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="pushButton_calc">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="spinBox_rate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QComboBox" name="comboBox_new_repl">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>One-time</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once a month</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once every 2 months</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once a quarter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Semiannually</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once a year</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox_period">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>1200</number>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QTableWidget" name="tableWidget_withdraw">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>0</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>70</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>0</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>24</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Index</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Period</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>None</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="11" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Result</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>541</width>
         <height>85</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,5,1">
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_total">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_charge">
          <property name="text">
           <string>Interest charges</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_charge">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_tex_amount">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_tax_amount">
          <property name="text">
           <string>Tax amount</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_total">
          <property name="text">
           <string>Deposit amount with interest</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_tex_amount_rub">
          <property name="text">
           <string>Rub</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_charge_rub">
          <property name="text">
           <string>Rub</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_total_rub">
          <property name="text">
           <string>Rub</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="pushButton_minus_repl">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox_amount">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>100000000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_peroid">
      <property name="text">
       <string>Placement period</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_amount">
      <property name="text">
       <string>Deposit amount</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_withdraw">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>List of partial withdrawals</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="pushButton_minus_withdraw">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_rate_percent">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>% / year</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_period_month">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Month</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboBox_capital">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>Add to contribution</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pay out</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="spinBox_new_repl">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <double>100000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_tax_percent">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>% / year</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_tax">
      <property name="text">
       <string>Tax rate</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox_freq">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>Every day</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Every week</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once a month</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once a quarter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Semiannually</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once a year</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>At the end of the term</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_rate">
      <property name="text">
       <string>Interest rate</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QComboBox" name="comboBox_new_withdraw">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>One-time</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once a month</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once every 2 months</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once a quarter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Semiannually</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once a year</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="pushButton_plus_repl">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_replen">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Replenishment list</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QDoubleSpinBox" name="spinBox_new_withdraw">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <double>100000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_frequency">
      <property name="text">
       <string>Payment frequency</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QTableWidget" name="tableWidget_repl">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>0</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>70</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>0</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>24</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Index</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Period</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>None</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="comboBox_amount">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Rub</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>USD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EUR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CNY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JPY</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
